dev-scripts/openshift_deploy.sh

```diff
@@ -131,6 +136,8 @@ elif [ "${OPENSHIFT_FLAVOR}" == "osio" ]; then
   CHE_KEYCLOAK_DISABLED=${CHE_KEYCLOAK_DISABLED:-${DEFAULT_CHE_KEYCLOAK_DISABLED}}
   DEFAULT_CHE_DEBUGGING_ENABLED="false"
   CHE_DEBUGGING_ENABLED=${CHE_DEBUGGING_ENABLED:-${DEFAULT_CHE_DEBUGGING_ENABLED}}
+  DEFAULT_OPENSHIFT_SKIP_TLS="false"
+  OPENSHIFT_SKIP_TLS=${OPENSHIFT_SKIP_TLS:-${DEFAULT_OPENSHIFT_SKIP_TLS}}
```

Comment:
That's more about OC than OPENSHIFT here. So I would rather call the variable `OC_SKIP_TLS`.

Reply:
@l0rd  Fixed